src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -115,19 +115,20 @@ public function syncPivot($model, $data)
 
```

Comment:
This is another place where we use `$data` - and the variable name tells us absolutely nothing about what's stored in it. This function is so SO old... I know. Time to update it to our latest standards, and `$data` doesn't cut it. In fact...

**New coding rule: `$data` is now banned as a variable name:** 
- we don't create any more code that uses `$data`;
- every time we touch a function that uses `$data` as a variable name or parameter... we rename it to something more explicit;
 
In this case, let's please rename it to `$input`. That's what it is.